app/schemas/federation_entities.json

```diff
@@ -0,0 +1,366 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://diasporafoundation.org/schemas/federation_entities-1.0.0.json",
```

Comment:
Gem names don't contain a `.` (see: http://guides.rubygems.org/name-your-gem/), and also I think it should contain `diaspora`. So `diaspora_federation_json_schema` or maybe `diaspora_federation-json_schema`?